openstack/nova/templates/_console_deployment.yaml.tpl

```diff
@@ -42,7 +42,7 @@ spec:
           name: nova-etc
       containers:
       - name: nova-console-{{ $name }}
-        image: {{ .Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-nova-{{ $name }}proxy:{{index .Values (print "imageVersionNova" (title $name) "proxy") | default .Values.imageVersion | required "Please set nova.imageVersion or similar" }}
```

Comment:
Unfortunately, this breaks the pipeline, as it cannot derive the image name without making a full template evaluation:

Reply:
You could match the tuple . "XYZ" instead?

Reply:
btw, what exactly breaks at the pipeline? I didn't expected that

Reply:
Could you separate this change out? There is no need to do this in one go.

Reply:
just for console images or for all?

Reply:
Hmm, let me see... maybe I can fix it in the template after all.

Reply:
Okay, I gave it a try:
While I can match on tuple, which image is actually behind the image cannot be derived without template evaluation and the knowledge of the actual value.

I would say, leave all the "container_image.*" templates out, and we can revisit how to make the deployment image agnostic another time.

Reply:
this will throwing us back in weeks

Reply:
What will take weeks here?

Reply:
cause I need to test it with queens and mitaka, this I cannot do in this week

Reply:
please point me to the broken pipeline. I don't understand the ruby template problem

Reply:
please recheck